src/tests/system/lib/sssd/utils/authentication.py

```diff
@@ -43,12 +46,34 @@ def test_example(client: Client, provider: GenericProvider, method: str):
                 assert client.auth.parametrize(method).password('tuser', 'Secret123')
     """
 
-    def __init__(self, host: MultihostHost) -> None:
+    def __init__(self, host: MultihostHost, fs: LinuxFileSystem) -> None:
         """
         :param host: Remote host.
         :type host: MultihostHost
         """
         super().__init__(host)
+        self.fs: LinuxFileSystem = fs
+
+        self.passkey: PasskeyAuthenticationUtils = PasskeyAuthenticationUtils(host, fs)
+        """
+        Test passkey registration and passkey authentication with umockdev-run.
+
+        .. code-block:: python
+            :caption: Example usage
+
+            @pytest.mark.topology(KnownTopology.IPA)
+            def test_example(client: Client, ipa: IPA):
+                user = ipa.user('user1').add()
+                passkey_str = "passkey:NUZMRUXIb/W8Ij1GqwCDHSCWxt/SxWxckwtQjLYi/X6Y1qZFB+HI8WO6khzAjzsz24" +\
+                              "8kHbaeAf9qfmqfCky1Jg==,MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEIasAa8ogjPCKXeA" +\
+                              "4KY3t0W3xBRmG+E4D+MNoRIAJrYuNLSYtAcOL7DCbIfgc+7c5Y4Mh/FzoEyeumKGYMoyTfg=="
+                user.passkey_add(passkey_str)
+
+                client.sssd.pam['pam_passkey_auth'] = 'true'
```

Comment:
can I use `--user-auth-type=passkey` while adding LDAP users also?

Reply:
> AFAIK, this line isn't needed anymore because `pam_passkey_auth` is already `true` by default. But I think you need to create the user in `IPA` with the following option `--user-auth-type=passkey`. Am I right @justin-stephenson

Correct, `pam_passkey_auth` is set to true by default with https://github.com/SSSD/sssd/pull/6628